Misc/reputations.json

```diff
@@ -183,7 +183,8 @@
         "DomainReputation"
       ],
       "formatScript": "ExtractDomainFromUrlAndEmail",
-      "fromVersion": "3.6"
+      "fromVersion": "3.6",
+      "toVersion": "5.4.9"
```

Comment:
From and to version doesn't work in this file so this expiration won't work

Reply:
shouldn't we also add the new indicator here?

Reply:
@ronykoz we don't want to support this file from 5.5 but to create a different file for every indicator.
@hirshbergshachar really? but we want only one domain indicator from 5.5.
So what can we do about the one that's in this file?

Reply:
You should remove it from this file, and add it as a separate file with those to and from versions. You can see the other indicator types for which we did the same

Reply:
@hirshbergshachar So we actually do support expiration from the file.
Added toVersion in the meantime, I will open a PR to delete this file and work only with a separate file to each indicator.